File: app/src/main/java/com/va/kotlintaste/constant/DBConstant.java
package com.va.kotlintaste.constant;

/**
 * Created by Junmeng.Chen on 2017/9/14.
 */

public final class DBConstant {

    public static final String PERSON_URI_STRING = "content://" + DBConstant.AUTHORITY_PERSON + "/" + DBConstant.PATH_PERSON;

    public static final String AUTHORITY_PERSON = "com.va.kotlintaste";

    public static final String PATH_PERSON = "person";

    public static final int DB_VERSION = 1;

    public static final String DB_NAME = "taste_db.db";

    public static final String TABLE_PERSON = "person";

    public static final String KEY_ID = "_id";

    public static final String KEY_NAME = "_name";

    public static final String KEY_WHERE = "_where";

}

File: app/src/main/java/com/va/kotlintaste/constant/DbConstant.kt
package com.va.kotlintaste.constant

/**
 *
 * @author Junmeng.Chen
 * @date 2017/9/14
 */

class DbConstant {

    companion object {

        val PERSON_URI_STRING = "content://" + DbConstant.AUTHORITY_PERSON + "/" + DbConstant.PATH_PERSON

        val AUTHORITY_PERSON = "com.va.kotlintaste"

        val PATH_PERSON = "person"

        val DB_VERSION = 1

        val DB_NAME = "taste_db.db"

        val TABLE_PERSON = "person"

        val KEY_ID = "_id"

        val KEY_NAME = "_name"

        val KEY_WHERE = "_where"
    }

}
